disnake/colour.py

```diff
@@ -127,6 +127,11 @@ def from_hsv(cls: Type[CT], h: float, s: float, v: float) -> CT:
         """Constructs a :class:`Colour` from an HSV tuple."""
         rgb = colorsys.hsv_to_rgb(h, s, v)
         return cls.from_rgb(*(int(x * 255) for x in rgb))
+    
+    @classmethod
+    def from_hex(cls: Type[CT], hex: str) -> CT:
+        """Constructs a :class:`Colour` from hex value."""
+        return cls.from_rgb(*tuple(int(hex.replace("#","")[i:i+2], 16) for i in (0, 2, 4)))
```

Comment:
Could probably get away with something simpler, like
```py
return cls(int(hex.strip("#"), 16))
```

Alternatively, are you aware that you can just do `disnake.Colour(0xFFFFFF)`, for example?

Reply:
you can do `disnake.Colour(0xFFFFFF)` too maybe but what if you are asking for user input in slash command or an argument in traditional command?

Reply:
`return cls(int(hex.strip("#"), 16))` wont work prob

Reply:
ow it did work, ig i didnt look at it properly before